app/src/main/java/com/example/composetemplate/presentation/activities/MainActivity.kt

```diff
@@ -14,7 +17,28 @@ class MainActivity : ComponentActivity() {
         enableEdgeToEdge()
         setContent {
             ComposeTemplateTheme {
-                Navigator().CreateAppNavigation()
+//                Navigator().CreateAppNavigation()
```

Comment:
Now I understand what you meant. If we don't want the changes in this file, why push it? Alternatively, we can push just the changes that we need.